src/west/_bootstrap/main.py

```diff
@@ -191,44 +180,108 @@ def init_bootstrap(directory, args):
     # Clone the west source code and the manifest into west/. Git will create
     # the west/ directory if it does not exist.
 
-    clone('west repository', args.west_url, args.west_rev,
+    clone('west repository', west_url, west_rev,
           os.path.join(directory, WEST_DIR, WEST))
 
-    clone('manifest repository', args.manifest_url, args.manifest_rev,
+    clone('manifest repository', manifest_url, manifest_rev,
           os.path.join(directory, WEST_DIR, MANIFEST))
 
     # Create an initial configuration file
 
-    config = configparser.ConfigParser()
+    config_path = os.path.join(directory, WEST_DIR, 'config')
+    update_conf(config_path, west_url, west_rev, manifest_url, manifest_rev)
+    print('=== Initial configuration written to {} ==='.format(config_path))
 
-    config['west'] = {
-        'remote': args.west_url,
-        'revision': args.west_rev
-    }
+    # Create a dotfile to mark the installation. Hide it on Windows.
 
-    config['manifest'] = {
-        'remote': args.manifest_url,
-        'revision': args.manifest_rev
-    }
+    with open(os.path.join(directory, WEST_DIR, WEST_MARKER), 'w') as f:
+        hide_file(f.name)
 
-    config_path = os.path.join(directory, WEST_DIR, 'config')
+    print('=== West initialized ===')
+
+
+def reinit(config_path, args):
+    '''
+    Reinitialize an existing installation.
+
+    Currently, this is just a shorthand for updating the west/config
+    configuration file.
+    '''
+    if args.base_url:
```

Comment:
Wonder if we could get rid of `-b`. It's basically just a shorthand to save a little bit of copy-paste now.

Might be possible to factor out something here anyway though.

Reply:
I think as a convenience to the user, but not something I'd defend to death.
Being able to do:

`west init -b https://github.com/mycompany`

is nice and short.

Reply:
As a user, I find this option valuable even if it is redundant. I originally did not want to implement it but I found myself using it all the time when pulling down wworkdirs from non-mainline places.

Reply:
Adunno... still feels a bit weird to me to have a special command just for when you happen to have two repositories with particular names and with the same URL prefix.

Not really worth arguing though.